vendor/brave-ios/Ads/BATAdNotification.h

```diff
@@ -7,15 +7,15 @@
 NS_ASSUME_NONNULL_BEGIN
 
 typedef NS_ENUM(NSInteger, BATAdsConfirmationType) {
-  BATAdsConfirmationTypeNone,       // = ads::ConfirmationType::kNone
-  BATAdsConfirmationTypeClick,      // = ads::ConfirmationType::kClicked
-  BATAdsConfirmationTypeDismiss,    // = ads::ConfirmationType::kDismissed
-  BATAdsConfirmationTypeView,       // = ads::ConfirmationType::kViewed
-  BATAdsConfirmationTypeLanded,     // = ads::ConfirmationType::kLanded
-  BATAdsConfirmationTypeFlagged,    // = ads::ConfirmationType::kFlagged
-  BATAdsConfirmationTypeUpvoted,    // = ads::ConfirmationType::kUpvoted
-  BATAdsConfirmationTypeDownvoted,  // = ads::ConfirmationType::kDownvoted
-  BATAdsConfirmationTypeConversion  // = ads::ConfirmationType::kConversion
+  BATAdsCUndefinedonfirmationType,  // = ads::ConfirmationType::kUndefined
```

Comment:
Small typo here (should be `BATAdsConfirmationTypeUndefined`) and this no longer matches proper Obj-C enum naming. Should be `{Prefix}{Name}{Case}`. Please revert to how it was previously as this will break Swift imports and the iOS build.

Reply:
Fixed